src/lib/api/feed-manip.ts

```diff
@@ -183,7 +179,14 @@ export class FeedTuner {
   ): FeedViewPostsSlice[] {
     // remove any replies without at least 2 likes
     for (let i = slices.length - 1; i >= 0; i--) {
-      if (slices[i].isFullThread || !slices[i].isReply) {
+      // I can't figure out how to make this work when there's a *single* reply
+      // to an older post in a thread. It's not "a thread" because it doesn't
+      // have more than one item in the slice. But it is a reply. I'm not sure
+      // how to reliably determine if something is a thread.
+      //
+      // isFullThread was always problematic, because frequently you don't have
+      // a root on an old thread
```

Comment:
I am not entirely confident about any of this. I'm not a react person, and couldn't get a debugger with sourcemaps working easily, so I was limited to `console.log()` 😉